modules/statics/src/coins.ts

```diff
@@ -109,6 +110,9 @@ export const coins = CoinMap.fromCoins([
   account('tstx', 'Testnet Stacks', Networks.test.stx, 6, UnderlyingAsset.STX),
   account('sol', 'Sol', Networks.main.sol, 9, UnderlyingAsset.SOL, SOL_FEATURES, KeyCurve.Ed25519),
   account('tsol', 'Testnet Sol', Networks.test.sol, 9, UnderlyingAsset.SOL, SOL_FEATURES, KeyCurve.Ed25519),
+  // TODO : Uncomment when core functionality is implemented
+  //  account('near', 'Sol', Networks.main.near, 9, UnderlyingAsset.NEAR, NEAR_FEATURES, KeyCurve.Ed25519),
+  //  account('tnear', 'Testnet Sol', Networks.test.near, 9, UnderlyingAsset.NEAR, NEAR_FEATURES, KeyCurve.Ed25519),
```

Comment:
NEAR uses 24 decimals for its smallest unit: `1 NEAR = 10^24 yoctoNEAR`
https://docs.near.org/docs/api/naj-quick-reference#near--yoctonear

```suggestion
  // TODO : Uncomment when core functionality is implemented
  //  account('near', 'Near', Networks.main.near, 24, UnderlyingAsset.NEAR, NEAR_FEATURES, KeyCurve.Ed25519),
  //  account('tnear', 'Testnet Near', Networks.test.near, 24, UnderlyingAsset.NEAR, NEAR_FEATURES, KeyCurve.Ed25519),
```